src/Debugger/Debuggee.php

```diff
@@ -292,6 +292,37 @@ public function updateBreakpoint(Breakpoint $breakpoint)
         ]);
     }
 
+    /**
+     * Set a breakpoint for this debuggee.
+     *
+     * Example:
+     * ```
+     * $breakpoint = $debuggee->setBreakpoint('DebuggerClient.php', 10);
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/debugger/api/reference/rest/v2/debugger.debuggees.breakpoints/set Breakpoint set API documentation.
+     * @codingStandardsIgnoreEnd
+     *
+     * @param string $path Path to the source file.
+     * @param int $line Line within the source file.
+     * @param array $options [optional] Array of Breakpoint constructor arguments. See
+     *        {@see Google\Cloud\Debugger\Breakpoint::__construct()} for configuration details.
+     */
+    public function setBreakpoint($path, $line, array $options = [])
+    {
+        $breakpoint = new Breakpoint([
+            'location' => [
+                'path' => $path,
+                'line' => $line
+            ]
+        ] + $options);
+        $resp = $this->connection->setBreakpoint([
```

Comment:
We need to allow for users to pass in request configuration options, like `timeout` and `restOptions`.

Reply:
I can add this to all of the debugger api requests. It looks like we don't actually document any of the request configuration options on each method.

I'm thinking to have the request options be a second optional argument (after the optional breakpoint fields).

Reply:
That's okay, we don't currently elsewhere yet either. The plan to execute on is something like the following, across the libraries:

`@param array $options [optional] Configuration options. For configuration options which apply to all network request, please see [here](/docs-for-common-options).`

> I'm thinking to have the request options be a second optional argument (after the optional breakpoint fields).

That sounds fair. Another decent option would be to nest it inside `$options`, which I believe we do elsewhere in the library. Use your judgement :).